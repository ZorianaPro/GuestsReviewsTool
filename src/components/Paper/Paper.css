.Paper {
  width: 100%;
  background-color: var(--colorWhite);
  box-shadow: 0 1px 1px 0 rgba(0,0,0,0.05), 0 3px 6px 0 rgba(0,0,0,0.05), 0 6px 12px 0 rgba(0,0,0,0.1);
  border-radius: var(--borderRadius);
  padding: 1em 2em;
}