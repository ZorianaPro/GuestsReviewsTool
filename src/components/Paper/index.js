import React from 'react';
import './Paper.css';

const Paper = ({
  children
}) => (
  <div className="Paper">
    { children }
  </div>
);

export default Paper;