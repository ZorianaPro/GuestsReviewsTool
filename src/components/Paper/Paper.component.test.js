import React from 'react';
import { shallow } from 'enzyme';
import Paper from './';

describe('Paper', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <Paper />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<Paper />)).toMatchSnapshot();
  });
});
