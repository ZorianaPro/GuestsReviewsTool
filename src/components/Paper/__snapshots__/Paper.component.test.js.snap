// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paper renders as expected 1`] = `
<div
  className="Paper"
/>
`;
